android-sdk/tasks/main.yml

```diff
@@ -75,12 +75,18 @@
     dest: /tmp/{{ config_file }}
     force: yes
 
+#-
```

Comment:
if we are going to keep this, let's add a comment there to explain why this is commented and why we want to keep them, and when we should uncomment these